packages/nodes-base/nodes/Wait/Wait.node.ts

```diff
@@ -420,12 +420,17 @@ export class Wait extends Webhook {
 
 			waitAmount *= 1000;
 
+			// Timezone does not change relative dates, since they are just
+			// a number of seconds added to the current timestamp
 			waitTill = new Date(new Date().getTime() + waitAmount);
 		} else {
-			// resume: dateTime
-			const dateTime = context.getNodeParameter('dateTime', 0) as string;
+			const dateTimeStr = context.getNodeParameter('dateTime', 0) as string;
 
-			waitTill = new Date(dateTime);
+			waitTill = DateTime.fromFormat(dateTimeStr, "yyyy-MM-dd'T'HH:mm:ss", {
```

Comment:
Fix timezone only for specific points in time, where 4pm is something in UK vs Berlin.